src/onnx/onnx.cpp

```diff
@@ -778,6 +779,51 @@ struct onnx_parser
         return add_bias(args, l1, 1);
     }
 
+    void
+    tune_padding_to_symmetric(int64_t& left, int64_t& right, const int stride, int64_t& s_start)
+    {
+        s_start = 0;
+        if(left > right)
+        {
+            right   = left;
+            s_start = 0;
```

Comment:
is this needed since `s_start` is set to 0 in L785?

Reply:
fixed